test/passing/js_source.ml.ref

```diff
@@ -1458,8 +1458,8 @@ let ty_abc : (([ `A of int | `B of string | `C ] as 'a), 'e) ty =
          ]
 
        method inj
-           : type c.  (int -> string -> noarg -> unit, c) ty_sel * c
-                   -> [ `A of int | `B of string | `C ] =
+             : type c.  (int -> string -> noarg -> unit, c) ty_sel * c
+                     -> [ `A of int | `B of string | `C ] =
```

Comment:
I didn't manage to get rid of this regression, it doesn't seem that negative to me in comparison to the other improvements.

Reply:
I tried as well but no success. `fmt_kind` deserves a little refactor to return something better than `l, eq, expr`. The `l` makes no sense to me.

Reply:
I agree, but maybe it can be fixed in another PR (as long as @jberdine @hhugo don't mind about this regression in the meantime)